app/src/userpages/components/StreamPage/List/index.jsx

```diff
@@ -288,13 +294,31 @@ class StreamList extends Component<Props, State> {
                             </thead>
                             <tbody>
                                 {streams.map((stream) => (
-                                    <tr key={stream.id}>
-                                        <th>{stream.name}</th>
-                                        <td title={stream.description}>{stream.description}</td>
-                                        <td>{moment(stream.lastUpdated).fromNow()}</td>
-                                        <td>-</td>
-                                        <td><StatusIcon /></td>
-                                        <td>
+                                    <tr
+                                        key={stream.id}
+                                        className={styles.streamRow}
```

Comment:
Adding a click handler to the `tr` doesn't work for the last cell that has the dropdown so the handlers are added to each `td`.

Reply:
@juhah perhaps if you add a click handler to the last cell like: `onClick={(event) => event.stopPropagation()}` this would stop the click event from bubbling to the `tr`'s click handler?